PhysicsTools/NanoAOD/plugins/TriggerOutputBranches.cc

```diff
@@ -7,18 +7,18 @@
 void 
 TriggerOutputBranches::updateTriggerNames(TTree & tree, const edm::TriggerNames & names, const edm::TriggerResults & triggers) 
 {  
-   std::vector<std::string> newNames(triggers.getTriggerNames());
-   if(newNames.empty()) {
-       for(unsigned int j=0;j<triggers.size();j++) {
-           newNames.push_back(names.triggerName(j));
-       }
-   }
+   std::vector<std::string> newNames(names.triggerNames());
```

Comment:
this doesn't work because we use it both for L1 fake triggerNames and HLT triggernames, in one case we get the names from getTriggerNames in the other we use the "names" object